src/events/message/messageCreate.ts

```diff
@@ -14,21 +14,9 @@ export default class extends Event {
 	}
 
 	public async run(message: Message): Promise<any> {
-		const guildCfg = await databaseManager.getConfig(message.guild as Guild);
-		await databaseManager.addToMessageToDB(message);
-
-		if (guildCfg === undefined || guildCfg.prefix === undefined) {
-			await databaseManager.deleteGuild(message.guild as Guild);
-			await databaseManager.getGuild(message.guild as Guild);
-			if (!(await databaseManager.getConfig(message.guild as Guild)))
-				return this.safeReply(message, "Please remove and re-add the bot to the server https://bulbbot.rocks/invite, there has been an error with the configuration of the guild");
+		if (!message.guild) {
+			return;
 		}
-
-		await AutoMod(this.client, message);
-	}
-
-	private async safeReply(message: Message, text: string): Promise<Message | undefined> {
-		if (!message.guild?.me?.permissionsIn(message.channel.id).has(Permissions.FLAGS.SEND_MESSAGES)) return;
-		return await message.channel.send(text);
+		await Promise.allSettled([databaseManager.addToMessageToDB(message), AutoMod(this.client, message)]);
```

Comment:
These are safely independent so we can start them together rather than making one wait for the other